Update entity_types.md (#30595)
DataDog/documentation: content/en/internal_developer_portal/software_catalog/entity_model/entity_types.md

{{% /tab %}}

{{% tab "Frontend" %}}

> In Software Catalog, a frontend (`kind:frontend`) represents a frontend application—such as a browser-based SPA or mobile app—that interacts with services and APIs. Frontend entities offer a structured way to model user-facing applications in the same catalog alongside backend services.

Reviewer: It'd probably help to spell out SPA. Most readers are technical and probably familiar with it, but it'd still help for clarity and readability.
```suggestion
In Software Catalog, a frontend (`kind:frontend`) represents a frontend application—such as a browser-based single-page application or mobile app—that interacts with services and APIs. Frontend entities offer a structured way to model user-facing applications in the same catalog alongside backend services.
```

### Example YAML definitions

> ```

Reviewer: This will enable syntax highlighting and the Copy button in the rendered docs:
```suggestion
```yaml
```

apiVersion: v3
kind: frontend
metadata:
name: "checkout-webapp"
displayName: "Checkout Web App"
description: Main frontend experience for the checkout flow in Shopist
owner: shopist-frontend
additionalOwners:
- name: ux-platform-team
type: team
links:
- name: "UX Design Guidelines"
type: doc
url: https://wiki.internal/checkout-design
- name: "Frontend Source Code"
type: repo
provider: github
url: https://github.com/shopist/checkout-webapp
spec:
type: browser
lifecycle: production
tier: tier1
dependsOn:
- service:checkout-api
- service:payment-service
componentOf:
- system:shopist-checkout-platform
```

When this definition is created:

> 1. The frontend app appears under the Frontend Apps section in Software Catalog.
> 2. If a RUM application exists with the same name, its telemetry is automatically linked.
> 3. The entity aggregates metadata, dependencies, and real-time RUM performance metrics in a unified view.

Reviewer: If these don't necessarily happen in a specific order, bullets are fine:
```suggestion
- The frontend app appears under the Frontend Apps section in Software Catalog.
- If a RUM application exists with the same name, its telemetry is automatically linked.
- The entity aggregates metadata, dependencies, and real-time RUM performance metrics in a unified view.
```
